javascripts/discourse/templates/components/dc-category-card.hbs

```diff
@@ -5,6 +5,9 @@
 >
   <div class="dc-card__content">
     <h2 class="dc-card__title dc-clamp-2">
+      {{#if category.read_restricted}}
```

Comment:
@arathjz is this the right attribute for a private category? looks weird to me it's named `read_restricted`